File: DSA/Shlok_590017851/day1_q3.c
#include <stdio.h>
int sumOfArray(int arr[], int n) 
{
    int sum = 0;
    for(int i = 0; i < n; i++) 
    {
        sum += arr[i];
    }
    return sum;
}
int main()
{
    int arr[] = {5, 10, 15, 20, 25};
    int n = sizeof(arr) / sizeof(arr[0]);

    int total = sumOfArray(arr, n);
    printf("Sum of array elements: %d\n", total);
    return 0;
}

File: DSA/Prabhat_500102136/Day1/Day1_Q3.java
import java.util.*;
class Day1_Q3 {
    public static int sum(int arr[],int n){
        int sum=0;
        for(int i=0;i<n;i++){
            sum+=arr[i];
        }
      return sum;
    }
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter the size of array");
        int n = scan.nextInt();
        System.out.println("Start entering elements of array");
        int arr[] = new int[n];
        for(int i=0;i<n;i++){
            arr[i]= scan.nextInt();
        }
        System.out.println("Sum of all elements "+sum(arr,n));
   
    }
}
